oregoninvasiveshotline/static/js/main.js

```diff
@@ -81,3 +81,40 @@ function getDefaultMapOptions(){
 
     return mapOptions;
 }
+
+// Code to allow the Hamburger Menu to be functional on small pages
```

Comment:
Since we replaced this hamburger menu with the other menu that opens the side bar menu, we could get rid of this. I have a pull-request that takes care of this later.